packages/forms/stories-core/layouts.js

```diff
@@ -1,9 +1,30 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { HeaderBar, Layout, Dialog, Drawer } from '@talend/react-components';
+import { action } from '@storybook/addon-actions';
+
 import Form from '../src/index';
 
 const simple = require('./json/concepts/core-simple.json');
+const old = require('../stories/json/simple.json');
```

Comment:
I can't figure out why (maybe coincidence), but the stories that use `old` data have the buttons left-aligned. Should they not be right-aligned?

Reply:
if you take the OLD stories you can see the submit btn is always left aligned ...